parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -238,13 +238,13 @@ function configAzureNetworkPolicy() {
     # Mirror from https://github.com/Azure/azure-container-networking/releases/tag/$AZURE_PLUGIN_VER/azure-vnet-cni-linux-amd64-$AZURE_PLUGIN_VER.tgz
     downloadUrl ${VNET_CNI_PLUGINS_URL} | tar -xz -C $CNI_BIN_DIR
     # Mirror from https://github.com/containernetworking/cni/releases/download/$CNI_RELEASE_VER/cni-amd64-$CNI_RELEASE_VERSION.tgz
-    downloadUrl ${CNI_PLUGINS_URL} | tar -xz -C $CNI_BIN_DIR ./loopback
+    downloadUrl ${CNI_PLUGINS_URL} | tar -xz -C $CNI_BIN_DIR ./loopback ./portmap
     chown -R root:root $CNI_BIN_DIR
     chmod -R 755 $CNI_BIN_DIR
 
-    # Copy config file
-    mv $CNI_BIN_DIR/10-azure.conf $CNI_CONFIG_DIR/
-    chmod 600 $CNI_CONFIG_DIR/10-azure.conf
+    # Copy config file. After azure cni is released, we should change the 10-azure.conf to 10-azure.conflist
+    mv $CNI_BIN_DIR/10-azure.conflist $CNI_CONFIG_DIR/
+    chmod 600 $CNI_CONFIG_DIR/10-azure.conflist
```

Comment:
This is dependent on Azure CNI drop that will have 10-azure.conflist file. We need to add the changes to pick up new drop of Azure CNI that has this file.

Reply:
CNI version 1.0.1 is now available. please update this PR to pick 1.0.1 from https://github.com/Azure/azure-container-networking/releases/tag/v1.0.1 

This change requires v1.0.1 and v1.0.1 requires this change.

@saiyan86 @jackfrancis

Reply:
See #2064, thanks for the notification @sharmasushant!